Ports/pt2-clone/patches/add-dummy-configure.patch

```diff
@@ -0,0 +1,7 @@
+diff --git a/configure b/configure
+new file mode 100644
+index 0000000..04bfb39
+--- /dev/null
++++ b/configure
+@@ -0,0 +1 @@
++#nop
```

Comment:
What's that for? If this doesn't have a configure script, just omit the `useconfigure=true` (defaults to false).

Reply:
I have it there so i can run cmake. Or is there some other way to run cmake without needing to use useconfigure?

Reply:
ohh, you're running cmake in the configure step, and that expects a configure script. Let's do this instead:

```diff
--- a/Ports/.port_include.sh
+++ b/Ports/.port_include.sh
@@ -151,6 +151,7 @@ func_defined pre_configure || pre_configure() {
     :
 }
 func_defined configure || configure() {
+    chmod +x "${workdir}"/"$configscript"
     run ./"$configscript" --host=i686-pc-serenity $configopts
 }
 func_defined build || build() {
@@ -252,7 +253,6 @@ do_configure() {
     if [ "$useconfigure" = "true" ]; then
         echo "Configuring $port!"
         pre_configure
-        chmod +x "${workdir}"/"$configscript"
         configure
     else
         echo "This port does not use a configure script. Skipping configure step.
```

when `configure()` is overridden we should not assume that `configscript` will be run or even exists.

Reply:
So this should be resolved now